code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -97,6 +97,22 @@ nobiliumsuppression = INFINITY
 			air.gases[/datum/gas/water_vapor][MOLES] -= MOLES_GAS_VISIBLE
 			. = REACTING
 
+//freon: helps the station chill out
+/datum/gas_reaction/freon_freeze
+	priority = 1
```

Comment:
Please check the other gases priorities for the number to use here

Reply:
This shouldn't cause any issues, as the water vapor turf application and freon turf application will happen in either order. There's no reaction including both freon and water vapor, so there's no chance of a loop of reactions bricking the atmospherics subsystem.

Reply:
Uhh plasmafire + Freon? Also I'd prefer not to risk, he'd just have to change the priority to an unused number

Reply:
These priorities should be using #defines so they're categorized somewhere.